packages/ui-core/shared/src/lib/table-components/employee-with-links/employee-with-links.component.ts

```
@@ -12,45 +12,33 @@ export class EmployeeWithLinksComponent implements OnInit {
 	@Input() rowData: any;
 	@Input() value: any;
 
-	employees: any[] = [];
+	displayedEmployees: any[] = [];
+	maxDisplay: number = 5;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)